packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -66,6 +67,7 @@ const resolveSideEffects = () => ({
 	redirect: Redirect.rewind(),
 	forbidden: Forbidden.rewind(),
 	notFound: NotFound.rewind(),
+	cookie: SetCookie.rewind(),
```

Comment:
You might want to also link to `https://github.com/gaearon/react-side-effect` in the comments above